cluster/gce/config-default.sh

```diff
@@ -182,7 +182,10 @@ RUNTIME_CONFIG="${KUBE_RUNTIME_CONFIG:-}"
 FEATURE_GATES="${KUBE_FEATURE_GATES:-ExperimentalCriticalPodAnnotation=true}"
 
 if [[ ! -z "${NODE_ACCELERATORS}" ]]; then
-    FEATURE_GATES="${FEATURE_GATES},Accelerators=true"
+    FEATURE_GATES="${FEATURE_GATES},DevicePlugins=true"
```

Comment:
how does switching this work across upgrades? Or if it doesn't that should be mentioned as part of the release note for this pr.

Reply:
`Accelerators` is an alpha feature. `DevicePlugins` is also an alpha feature (and the replacement for `Accelerators`).

I had added the following release note to the PR:
```
GCE nodes with NVIDIA GPUs attached now expose `nvidia.com/gpu` as a resource instead of `alpha.kubernetes.io/nvidia-gpu`.
```
which captures the difference between clusters created using the old script and clusters created using the new script. But I haven't actually tried upgrading a cluster that had the old flag to a cluster with the new flag. In GKE, we don't have to worry about this because we don't allow alpha cluster upgrade.

What should be the right release note here?
cc - @vishh

Reply:
Upgrades are not really supported when alpha features are turned on. So I don't see much value in thinking about the upgrade/downgrade scenario. We recommend users to stick to specific versions and our workflow has changed considerably over releases while in alpha.

Reply:
I think the release note as it is LGTM

Reply:
@roberthbailey , I tried the following:

```
# Last commit from master on this branch
git checkout 55e216f56eac0082acc6be655d9ae09cf9ba38a8

go run hack/e2e.go -- -v --build

export NODE_ACCELERATORS=type=nvidia-tesla-k80,count=2; export KUBE_NODE_OS_DISTRIBUTION=gci; export KUBE_GCE_ZONE=us-west1-b; export KUBE_GCE_NODE_IMAGE=gke-1-8-2-gke-0-cos-stable-60-9592-90-0-v171103-pre-nvda-gpu; export KUBE_GCE_NODE_PROJECT=gke-node-images;
cluster/kube-up.sh

# Latest commit on this branch
git checkout cf292754ba423aa6782564ea83fe48cc1ed677d4

go run hack/e2e.go -- -v --build

export NODE_ACCELERATORS=type=nvidia-tesla-k80,count=2; export KUBE_NODE_OS_DISTRIBUTION=gci; export KUBE_GCE_ZONE=us-west1-b; export KUBE_GCE_NODE_IMAGE=gke-1-8-2-gke-0-cos-stable-60-9592-90-0-v171103-pre-nvda-gpu; export KUBE_GCE_NODE_PROJECT=gke-node-images;
cluster/gce/upgrade.sh -l
```

Master got the new label `cloud.google.com/gke-accelerator=nvidia-tesla-k80`. Master also had the correct feature-gate `DevicePlugins=true` set. However, not sure how to check node upgrade because upgrade.sh doesn't support upgrading nodes to local binaries.